### PR TITLE
068 team member modal

### PR DESCRIPTION
**Pull request by jessicazeng13**
## Context

Closes 068 team member modal

## What Changed?

Minor styling changes, changed current reach implementation to next syntax + debugging. Couldn't get the photo to show up though, so we hardcoded the image url.

## How To Review

Just the Team.tsx file, I kept the old implementation as a comment in one of the instances of images. 

## Notes

Hardcoded the image, something about the activeMember.Photo attribute is broken. Need to fix at a later stage.

### CODE REVIEW

File: next/util/image.ts [hunk 1]
```diff
@@ -3,5 +3,6 @@ import { ImageType } from "@/schemas/Image";
 export const getLargestImageUrl = (img: ImageType) => {
   const { large, medium, small, thumbnail } = img.data.attributes.formats;
   const largestFormat = large ?? medium ?? small ?? thumbnail ?? { url: "" };
+  console.log("myurl", `${process.env.STRAPI_URL}${largestFormat.url}`); // Delete this late, this is for debugging
```
Comment (YvonneLiew) [inline next/util/image.ts]: Could you please delete this print statement?

**Review by YvonneLiew — CHANGES_REQUESTED** [review]

File: next/util/image.ts (hunk 1 above)
Comment (jessicazeng13) [reply next/util/image.ts:6]: oh that print statement isnt there for me?

Comment (jessicazeng13) [reply next/util/image.ts]: done!

**Review by YvonneLiew — APPROVED** [review]
lgtm!

**Review by Oculux314 — APPROVED** [review]
@jessicazeng13 Deiza's on holiday so I'll jump on this review - it looks great!

I have a suspicion about this image import actually - but will leave till later to check it out properly, for now happy to merge this 👍

**Pull request merged**